.github/workflows/release.yml

```
@@ -123,7 +123,7 @@ jobs:
           git config --global user.name "release[bot]"
           git config --global user.email "release[bot]@users.noreply.github.com"
           git add version.txt
-          git commit -m "chore: update stamping version to ${{ inputs.version }}"
+          git diff --quiet && git diff --staged --quiet || git commit -m "chore: update version.txt to ${{ inputs.version }}"
```

shellcheck doesn't like this. Since we `git add` the file above, we can simply

```suggestion
          git diff --staged --quiet || git commit -m "chore: update version.txt to ${{ inputs.version }}"
```